src/v/cluster/security_manager.cc

```diff
@@ -108,6 +123,60 @@ do_apply(create_user_cmd cmd, security::credential_store& store) {
     return errc::success;
 }
 
+/*
+ * handle: rename role command
+ */
+static std::error_code
+do_apply(rename_role_cmd cmd, security::role_store& store) {
+    if (store.contains(cmd.value)) {
+        return errc::role_exists;
+    }
+    auto role = store.get(cmd.key);
+    auto removed = store.remove(cmd.key);
+    if (!role || !removed) {
+        return errc::role_does_not_exist;
+    }
+    // TODO(oren): The old role has been removed at this point, so any failure
+    // would be due to an issue with the underlying container. Is that enough of
+    // a guarantee to ignore the result here?
```

Comment:
What failures are you talking about? Does this comment also apply in `update_role_cmd` and `create_role_cmd`?

I am not sure this comment is high value.

Reply:
done